source/index.html.md

```diff
@@ -50,17 +43,99 @@ $inspetor = new InspetorClient($inspetor_config);
 ?>
 ```
 
-To setup an Inspetor Library in any language you first need to set some configuration. 
+The first step of integrating Inspetor's antifraud services into your product is instantiating the library.
+
+In order to do so, you will need to pass the `InspetorClient` your product's **App ID** and **Tracker Name**, both of which will be provided to you by the Inspetor Team.
+
+_Et voilà_! You now have an Inspetor client object that is capable of sending all of the information necessary to teach Inspetor's decision layer how to prevent fraud at your company. However, there is obviously a difference between instantiating a client instance and using it properly.
```

Comment:
In 'However, there is obviously a difference between instantiating a client instance and using it properly.' would remove 'obviously'